otel/conf.go

```diff
@@ -54,7 +55,7 @@ service:
 {{- end}}
 {{define "hostmetrics" -}}
   hostmetrics/{{.HostMetricsID}}:
-    collection_interval: {{.CollectionInterval | validateCollectionInterval "hostmetrics"}}
```

Comment:
Nice catch! Is this worth a separate PR though? Seems a bit unrelated to `Fix user agent strings and the otel uptime version label.`, right?

Reply:
Ah. This is part of synchronizing the error messages across collectd and otel… I'm happy to mention that in the PR description if needed.